vendors/circom/README.md

```diff
@@ -81,3 +81,11 @@ Optional arguments:
 --disable_fast_twiddles_mode
                     Disables fast twiddle mode on Icicle NTT domain initialization.
 ```
+
+## How to compile witness generator
+
+```shell
+bazel build //circomlib/build/...
+bazel-bin/circomlib/build/compile_witness_generator <full_path_to_large_cpp>
+# and you will have witness_generator binary like when you use circom but this will compile much faster.
```

Comment:
For clarity and better flow, perhaps moving this line to right after line 85 as an introductory sentence would be better. + changing it to something like "This creates a witness_generator binary similar to when circom is used, but compiles much faster in comparison."